doc/build-unix.md

```diff
@@ -285,6 +285,50 @@ or building and depending on a local version of Berkeley DB 4.8. The readily ava
 As mentioned above, when maintaining portability of the wallet between the standard Dogecoin Core distributions and independently built
 node software is desired, Berkeley DB 4.8 must be used.
 
+Setup and Build Example: NixOS
+------------------------------
+This example lists the steps necessary to setup and build a full GUI distribution of the latest changes on NixOS.
+
+Clone and enter the repo:
+
+    $ git clone https://github.com/dogecoin/dogecoin
+    $ cd dogecoin
+
+Creating a file in the root of the repository called `shell.nix` with the following contents:
+
+```nix
+{ pkgs ? import <nixpkgs> {} }:
+  with pkgs; mkShell {
+    nativeBuildInputs = [
+      pkg-config
+      autoreconfHook
+      openssl
+      db5
+      util-linux
+      boost
+      zlib
+      libevent
+      miniupnpc
+      qt4
+      protobuf
+      qrencode
+    ];
+}
+```
```

Comment:
I'm wondering, could this be a file that's added to the repo? As in, make a file `contrib/nixos/shell.nix` with that in it? Helps keep the doc short.

Reply:
Nice idea! Just made that change.